### ISSUE TITLE
Known Bugs/Issues

### ISSUE THREAD

**Issue by DarkWav** [DarkWav/SAC#65]
### SAC currently has some known bugs:
* **Any vehicles including elytra and mounts are not supported. Minecarts DO work.**
* **AntiJesus checks missing.**
* **AntiClimb can be bypassed.**
* **No Checks against HighJump/HighStep.**
* **Slow falling will not be supported immideately after its implementation.**

More will be added here

**Comment by MalakasPlayzMC** [comment]
About the speed. You should check if the player has a speed potion. If the player has its speed level over than 2, disable the check for this player or set the  Speed-Threshold to a big number (eg. 638 because if for speed 2 we use threshold 5 on 255 (max speed on mcpe) what speed are we going to use? y=255x5:2=>y=637.5=> y~638). Its a temp solution that can be bypassed only if a hacker see that.

**Comment by DarkWav** [comment]
Yes, i will probably add speed support up to level 4 and then disable the check entirely. Thanks for your Ideas!

**Comment by DarkWav** [comment]
Fixed with commit 2df0ba5962de79f8194c02269949140884e3778f. Thanks for your idea!
You can grab the latest build from github or download the signed Phar [here](https://github.com/DarkWav/SAC/wiki/beta_RX15683472163547356412785412783451264126341268345123641251236672384623856238901341851)

**Comment by ghost** [comment]
We can't have false damage hack, so instant kill detection is useless

**Comment by DarkWav** [comment]
InstantKill is a weird name for the check, i know. Its actually pretty uselss right now since it only checks for
too fast hit speeds :) I will rename it soon.

**Comment by nikipuh** [comment]
AntiForceOP can be bypassed by having permission "*".

**Comment by DarkWav** [comment]
Thanks for Bypass reporting!

AntiForceOP is a tool to prevent ForceOP on your server.
Because of the way its coded, it can't block everything.
The bypass you described cant be fixed because of the way AntiForceOP works right now, maybe
i'll patch it one day.

### **If you are having critical ForceOP problems / Experience ForceOP cheaters on your Server, please report the Hacked Client to PocketMine-MP because the ForceOP hack most likely abuses a vulnurability in PocketMine-MP which can be fixed by the PocketMine team.**

If you have any further questions, feel free to ask them.

**Comment by DarkWav** [comment]
@nikipuh AntiForceOP has been reworked in version 3.3.8 so it cant't be bypassed with the permission "*" any more. Thanks for your idea!

**Comment by Creeperface01** [comment]
but fast hiting is already handled in pocketmine. You can't hit faster than 2x per second

**Comment by DarkWav** [comment]
Okay, good to know, this AC is quite old as of now, back at its development stages it was still a problem.
Its Architecture is horrible and it probably needs to be re-written entirely. Im not certain which API will be chosen for the re-write, tho.
